src/mono/mono/sgen/sgen-gc.c

```diff
@@ -4083,4 +4084,96 @@ sgen_check_canary_for_object (gpointer addr)
 	}
 }
 
+
+const guint MEM_PRESSURE_COUNT = 4;
```

Comment:
all the code around here doesn't use mono code style

Reply:
I think I got them all.